mods/content/fantasy/items/clothing/_loadout.dm

```diff
@@ -288,6 +288,18 @@
 		"white wine" =  /decl/material/liquid/ethanol/wine/premium,
 	))
 
+/decl/loadout_option/fantasy/utility/crutch
+	name = "crutch"
+	path = /obj/item/crutch/wooden/padded
+	available_materials = list(
```

Comment:
might be worth letting you pick a padding material (cotton, wool, leather, etc) or just adding an unpadded crutch as a pathtweak. currently it's always got leather padding